src/script.js

```diff
@@ -200,6 +209,176 @@
 			a.remove();
 		});
 
+		        var collapser = (function() {
+            
+        //function to check if only one button of the all is is 'ticked'
+            
+            function onlyOneIsTicked() {
```

Comment:
function name has to be made more clear. In the context of the app, this function would make no sense about what is that thing which is ticked.